lms/djangoapps/bulk_email/tests/test_models.py

```diff
@@ -52,14 +60,28 @@ def test_creation_with_optional_attributes(self):
         self.assertEquals(email.from_addr, from_addr)
 
     def test_bad_to_option(self):
-        course_id = SlashSeparatedCourseKey('abc', '123', 'doremi')
+        course_id = CourseKey.from_string('abc/123/doremi')
         sender = UserFactory.create()
         to_option = "fake"
         subject = "dummy subject"
         html_message = "<html>dummy message</html>"
         with self.assertRaises(ValueError):
             CourseEmail.create(course_id, sender, to_option, subject, html_message)
 
+    def test_cohort_target(self):
+        course_id = CourseKey.from_string('abc/123/doremi')
+        sender = UserFactory.create()
+        to_option = 'cohort:test cohort'
+        subject = "dummy subject"
+        html_message = "<html>dummy message</html>"
+        CourseCohort.create(cohort_name='test cohort', course_id=course_id)
+        email = CourseEmail.create(course_id, sender, [to_option], subject, html_message)
+        self.assertEquals(len(email.targets.all()), 1)
+        target = email.targets.all()[0]
+        self.assertEquals(target.target_type, SEND_TO_COHORT)
+        self.assertEquals(target.short_display(), 'cohort-test cohort')
+        self.assertEquals(target.long_display(), 'Cohort: test cohort')
```

Comment:
Nit: The transitive spellings (`assertEquals()`, as in x equals y) are deprecated in favor of the intransitive forms (`assertEqual()`, as in x and y are equal) in the latest versions of python.  https://docs.python.org/3.5/library/unittest.html#deprecated-aliases  Use the `s`-less version instead.